operations/security_and_authentication/Audit_Logs.md

```diff
@@ -0,0 +1,20 @@
+
+# Audit Logs
+
+How to enable/disable audit logging for host and Kubernetes APIs.
+
+Audit logs are used to monitor the system and search for suspicious behavior.
+Host and Kubernetes API audit logging can be enabled to produce extra audit logs for analysis.
+Enabling audit logging is optional and will generate some load and data on the non-compute nodes \(NCNs\).
+Host and Kubernetes API audit logging is not enabled by default.
+
+To enable or disable host and Kubernetes API audit logging, change the CSI setting:
```

Comment:
Example commands should be given for these. And if by CSI this means the `csi` binary used during installs, then instructions should also be provided for how users can access that post-install (since it is not included on the NCNs themselves).

Reply:
Amended with example commands and instructions to install csi.